ui/src/app/components/cases/list.component.html

```diff
@@ -213,7 +213,7 @@
           <div class="empty-run-md"></div>
           <div
             class="empty-text"
-            [translate]="!!query ? 'message.common.search.not_found' : 'testcase.list.not_found' | translate : { categoryType: isStepGroup? 'Step Groups' : 'Test Cases'}"></div>
+            [translate]="!!query ? 'message.common.search.not_found' : 'testcase.list.not_found' | translate : { categoryType: isStepGroup? 'Step Groups' : 'Test Cases',type: isDeletedFilter? 'added to Trash': 'created'}"></div>
```

Comment:
Trash can be in full lower case @nixonakash01

Reply:
okay bro, done